ckan/cli/notify.py

```diff
@@ -1,23 +1,48 @@
 # encoding: utf-8
 
+from logging import getLogger
 import click
 from ckan.model import Session, Package, DomainObjectOperation
 from ckan.model.modification import DomainObjectModificationExtension
+from ckan.logic import NotAuthorized, ValidationError
 
 
-@click.group(
-    name=u'notify',
-    short_help=u'Send out modification notifications.'
-)
+log = getLogger(__name__)
+
+
+@click.group(name="notify", short_help="Send out modification notifications.")
 def notify():
     pass
 
 
-@notify.command(
-    name=u'replay',
-    short_help=u'Send out modification signals.'
-)
+@notify.command(name="replay", short_help="Send out modification signals.")
 def replay():
     dome = DomainObjectModificationExtension()
     for package in Session.query(Package):
         dome.notify(package, DomainObjectOperation.changed)
+
+
+@notify.command(name="send_emails", short_help="Send out Email notifications.")
+def send_emails():
+    """
+    As currently implemented, it will only send notifications from dashboard
+    activity list.
+
+    It will send emails to users who have
+    `activity_streams_email_notifications` set in their profile.
+
+    It will send emails with updates depending on how
+    `ckan.email_notifications_since` is configured.
+    default: 2 days.
+    """
```

Comment:
```suggestion
    """ Sends an email to users notifying about new activities.
    
    As currently implemented, it will only send notifications from dashboard
    activity list if users have `activity_streams_email_notifications` set 
    in their profile. It will send emails with updates depending 
    on the `ckan.email_notifications_since` config. (default: 2 days.)
    """
```